app/models/person.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
+# person refers to anyone in the directory
 class Person < ApplicationRecord
   belongs_to :user, optional: true
+  has_many :requests, foreign_key: :requester_id
```

Comment:
Do you want to define the opposite side of the relationship too, or shall we wait for future feature work?